test/new-e2e/tests/process/docker_test.go

```diff
@@ -0,0 +1,260 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package process
+
+import (
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/DataDog/test-infra-definitions/components/datadog/dockeragentparams"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/DataDog/datadog-agent/test/fakeintake/aggregator"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	awsdocker "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/docker"
+)
+
+type dockerTestSuite struct {
+	e2e.BaseSuite[environments.DockerHost]
+}
+
+func TestDockerTestSuite(t *testing.T) {
+	agentOpts := []dockeragentparams.Option{
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can add `t.Parallel()` to your `Test*` to run them concurrently on the CI
```suggestion
	t.Parallel()
	agentOpts := []dockeragentparams.Option{
```